### DISCUSSION TITLE
How to mount Dropbox

### DISCUSSION THREAD

**Discussion by TheRedM** [AzuraCast/AzuraCast#4412] [Q&A]
I followed this guide:

https://docs.azuracast.com/en/user-guide/storage-locations/dropbox-configuration

But what i couldn't figure out is the Path/Suffix in AzuraCast, let's say my app name is Test and inside it there is a folder called Music, is the path going to be ```/Test/Music``` or ```/Test_Music```i tried everything i could think of nothing worked, and if i wanted to mount it in a specific station how do i specify that?

**Comment by subtleazn** [comment]
Im figuring this out also. Did anyone knows what is the proper paths for using dropbox?

**Comment by ItsMitchh** [comment]
Can you open a issue @TheRedM / @aniphreak so I can properly review the issue and see what we can to do fix? Cheers
